### PR TITLE
Add collectd + log_collector + Grafana support

### PR DESCRIPTION
**Pull request by simonpasquier**

### CODE REVIEW

**Review by elemoine — APPROVED** [review]
Looking good to me.

File: glusterfs/meta/heka.yml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  decoder:
+    glusterfs:
+      engine: sandbox
+      module_file: /usr/share/lma_collector/decoders/glusterfs.lua
+      module_dir: /usr/share/lma_collector/common;/usr/share/heka/lua_modules
+      adjust_timezone: true
+  input:
+{% if pillar.glusterfs.server is defined %}
+{% from "glusterfs/map.jinja" import server with context %}
+{%- if server.enabled %}
```
Comment (elemoine) [inline glusterfs/meta/heka.yml]: You use `cilent.get('enabled', False)` below, don't you want to use the same pattern here?

**Review by elemoine — CHANGES_REQUESTED** [review]
Small comment, otherwise looks good to me.

**Comment by elemoine** [comment]
Your PR title has gotten a bit off.

File: glusterfs/meta/heka.yml (hunk 1 above)
Comment (simonpasquier) [reply glusterfs/meta/heka.yml]: Done

**Review by elemoine — APPROVED** [review]

**Pull request merged**